apps/router-e2e/__e2e__/native-tabs/app/_layout.tsx

```diff
@@ -15,52 +15,74 @@ export default function Layout() {
     <ThemeProvider value={DarkTheme}>
       <ActiveTabsContext.Provider value={{ activeTabs, setActiveTabs }}>
         <NativeTabs
-          style={{
-            // fontSize: 16,
-            // fontWeight: 700,
-            // fontStyle: 'italic',
-            // fontFamily: 'Courier New',
-            backgroundColor: Platform.OS === 'android' ? 'black' : undefined,
-            // badgeBackgroundColor: 'green',
-            // color: 'blue',
-            tintColor: 'orange',
-            blurEffect: 'systemChromeMaterial',
-            labelVisibilityMode: 'auto',
-            rippleColor: 'orange',
-            iconColor: Platform.OS === 'android' ? '#888' : undefined,
-            color: Platform.OS === 'android' ? '#888' : undefined,
-            '&:active': {
-              fontSize: 14,
-              indicatorColor: Platform.OS === 'android' ? 'black' : undefined,
-            },
-          }}
-          minimizeBehavior="onScrollDown">
-          <NativeTabs.Trigger
-            name="index"
-            options={{
-              icon: { sf: 'applewatch.side.right', drawable: 'ic_phone' },
-              title: 'My Watch',
-            }}
-          />
+        // Both platforms
+        // labelStyle={{
+        //   fontSize: 16,
+        //   fontWeight: 700,
+        //   fontStyle: 'italic',
+        //   // fontFamily: 'Courier New',
+        //   color: Platform.OS === 'android' ? '#888' : undefined,
+        // }}
+        // backgroundColor={Platform.OS === 'android' ? 'black' : undefined}
+        // badgeBackgroundColor="green"
+        // tintColor="orange"
+        // iconColor={Platform.OS === 'android' ? '#888' : undefined}
+        // iOS only
+        // blurEffect="systemDefault"
+        // minimizeBehavior="onScrollDown"
+        // disableTransparentOnScrollEdge
+        // Android only
+        // labelVisibilityMode="auto"
+        // rippleColor="orange"
+        // indicatorColor="black"
```

Comment:
Commenting styles to see the default tab bar style. Uncomment to test specific options